cmd/botkube/main.go

```diff
@@ -20,85 +20,117 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
+	"github.com/infracloudio/botkube/pkg/filterengine"
+
+	"golang.org/x/sync/errgroup"
+
 	"github.com/infracloudio/botkube/pkg/bot"
 	"github.com/infracloudio/botkube/pkg/config"
 	"github.com/infracloudio/botkube/pkg/controller"
 	"github.com/infracloudio/botkube/pkg/log"
-	"github.com/infracloudio/botkube/pkg/metrics"
 	"github.com/infracloudio/botkube/pkg/notify"
 	"github.com/infracloudio/botkube/pkg/utils"
+
+	"github.com/infracloudio/botkube/pkg/metrics"
 )
 
 const (
 	defaultMetricsPort = "2112"
 )
 
+// TODO:
+// 	- Use context to make sure all goroutines shutdowns gracefully: https://github.com/infracloudio/botkube/issues/220
+//  - Make the code testable (shorten methods and functions, and reduce level of cyclomatic complexity): https://github.com/infracloudio/botkube/issues/589
```

Comment:
```suggestion
// TODO:
//  - Use context to make sure all goroutines shutdowns gracefully: https://github.com/infracloudio/botkube/issues/220
//  - Make the code testable (shorten methods and functions, and reduce level of cyclomatic complexity): https://github.com/infracloudio/botkube/issues/589
```

Reply:
TBH I used tabs because then IDE (like IntelliJ/GoLand) knows that this is a continuation of the "TODO" comment above and highlights it in the same way as TODO:
![Screenshot 2022-05-31 at 09 28 04](https://user-images.githubusercontent.com/7155799/171116763-11164a2d-9f17-4150-bffd-993697ffdd9e.png)